ext/mbstring/libmbfl/filters/cp932_table.h

```diff
@@ -29,67 +29,77 @@
- 0xD8F4,0xD9C6,0xD9C8,0xD9D1,0xF4E0,0xF4E1,
+	/* ku 117 */
+	0xC7CB, 0xC7CF, 0xC7D9, 0xF4CB, 0xF4CC, 0xC7E6, 0xC7EE, 0xC7FC,
+	0xC7EB, 0xC7F0, 0xC8B1, 0xC8E5, 0xC8F8, 0xC9A6, 0xC9AB, 0xC9AD,
+	0xF4CD, 0xC9CA, 0xC9D3, 0xC9E9, 0xC9E3, 0xC9FC, 0xC9F4, 0xC9F5,
+	0xF4CE, 0xCAB3, 0xCABD, 0xCAEF, 0xCAF1, 0xCBAE, 0xF4CF, 0xCBCA,
+	0xCBE6, 0xCBEA, 0xCBF0, 0xCBF4, 0xCBEE, 0xCCA5, 0xCBF9, 0xCCAB,
+	0xCCAE, 0xCCAD, 0xCCB2, 0xCCC2, 0xCCD0, 0xCCD9, 0xF4D0, 0xCDBB,
+	0xF4D1, 0xCEBB, 0xF4D2, 0xCEBA, 0xCEC3, 0xF4D3, 0xCEF2, 0xB3DD,
+	0xCFD5, 0xCFE2, 0xCFE9, 0xCFED, 0xF4D4, 0xF4D5, 0xF4D6, 0xF4D7,
+	0xD0E5, 0xF4D8, 0xD0E9, 0xD1E8, 0xF4D9, 0xF4DA, 0xD1EC, 0xD2BB,
+	0xF4DB, 0xD3E1, 0xD3E8, 0xD4A7, 0xF4DC, 0xF4DD, 0xD4D4, 0xD4F2,
+	0xD5AE, 0xF4DE, 0xD7DE, 0xF4DF, 0xD8A2, 0xD8B7, 0xD8C1, 0xD8D1,
+	0xD8F4, 0xD9C6, 0xD9C8, 0xD9D1, 0xF4E0, 0xF4E1,
 
- /* ku 118 */
- 0xF4E2,0xF4E3,0xF4E4,0xDCD3,0xDDC8,0xDDD4,0xDDEA,0xDDFA,
- 0xDEA4,0xDEB0,0xF4E5,0xDEB5,0xDECB,0xF4E6,0xDFB9,0xF4E7,
- 0xDFC3,0xF4E8,0xF4E9,0xE0D9,0xF4EA,0xF4EB,0xE1E2,0xF4EC,
- 0xF4ED,0xF4EE,0xE2C7,0xE3A8,0xE3A6,0xE3A9,0xE3AF,0xE3B0,
- 0xE3AA,0xE3AB,0xE3BC,0xE3C1,0xE3BF,0xE3D5,0xE3D8,0xE3D6,
- 0xE3DF,0xE3E3,0xE3E1,0xE3D4,0xE3E9,0xE4A6,0xE3F1,0xE3F2,
- 0xE4CB,0xE4C1,0xE4C3,0xE4BE,0xF4EF,0xE4C0,0xE4C7,0xE4BF,
- 0xE4E0,0xE4DE,0xE4D1,0xF4F0,0xE4DC,0xE4D2,0xE4DB,0xE4D4,
- 0xE4FA,0xE4EF,0xE5B3,0xE5BF,0xE5C9,0xE5D0,0xE5E2,0xE5EA,
- 0xE5EB,0xF4F1,0xF4F2,0xF4F3,0xE6E8,0xE6EF,0xE7AC,0xF4F4,
- 0xE7AE,0xF4F5,0xE7B1,0xF4F6,0xE7B2,0xE8B1,0xE8B6,0xF4F7,
- 0xF4F8,0xE8DD,0xF4F9,0xF4FA,0xE9D1,0xF4FB,
+	/* ku 118 */
+	0xF4E2, 0xF4E3, 0xF4E4, 0xDCD3, 0xDDC8, 0xDDD4, 0xDDEA, 0xDDFA,
+	0xDEA4, 0xDEB0, 0xF4E5, 0xDEB5, 0xDECB, 0xF4E6, 0xDFB9, 0xF4E7,
+	0xDFC3, 0xF4E8, 0xF4E9, 0xE0D9, 0xF4EA, 0xF4EB, 0xE1E2, 0xF4EC,
+	0xF4ED, 0xF4EE, 0xE2C7, 0xE3A8, 0xE3A6, 0xE3A9, 0xE3AF, 0xE3B0,
+	0xE3AA, 0xE3AB, 0xE3BC, 0xE3C1, 0xE3BF, 0xE3D5, 0xE3D8, 0xE3D6,
+	0xE3DF, 0xE3E3, 0xE3E1, 0xE3D4, 0xE3E9, 0xE4A6, 0xE3F1, 0xE3F2,
+	0xE4CB, 0xE4C1, 0xE4C3, 0xE4BE, 0xF4EF, 0xE4C0, 0xE4C7, 0xE4BF,
+	0xE4E0, 0xE4DE, 0xE4D1, 0xF4F0, 0xE4DC, 0xE4D2, 0xE4DB, 0xE4D4,
+	0xE4FA, 0xE4EF, 0xE5B3, 0xE5BF, 0xE5C9, 0xE5D0, 0xE5E2, 0xE5EA,
+	0xE5EB, 0xF4F1, 0xF4F2, 0xF4F3, 0xE6E8, 0xE6EF, 0xE7AC, 0xF4F4,
+	0xE7AE, 0xF4F5, 0xE7B1, 0xF4F6, 0xE7B2, 0xE8B1, 0xE8B6, 0xF4F7,
+	0xF4F8, 0xE8DD, 0xF4F9, 0xF4FA, 0xE9D1, 0xF4FB,
 
- /* ku 119 */
- 0xE9ED,0xEACD,0xF4FC,0xEADB,0xEAE6,0xEAEA,0xEBA5,0xEBFB,
- 0xEBFA,0xF4FD,0xECD6,0xF4FE
+	/* ku 119 */
+	0xE9ED, 0xEACD, 0xF4FC, 0xEADB, 0xEAE6, 0xEAEA, 0xEBA5, 0xEBFB,
+	0xEBFA, 0xF4FD, 0xECD6, 0xF4FE
 };
 
-static const int cp932ext3_eucjp_table_size = (sizeof (cp932ext3_eucjp_table) / sizeof (unsigned short));
+static const int cp932ext3_eucjp_table_size = (sizeof(cp932ext3_eucjp_table) / sizeof(unsigned short));
 
 #endif /* CP932_TABLE_H */
+
+/*
+ * Local variables:
+ * tab-width: 4
+ * c-basic-offset: 4
+ * indent-tabs-mode: t
+ * End:
+ * vim600: sw=4 ts=4 fdm=marker
+ * vim<600: sw=4 ts=4
+ */
```

Comment:
Hello, thanks for the patch, missing EOL character... (just hit enter at this file).